controller/src/main/java/com/inventory/controllers/API_PATH.java

```diff
@@ -2,8 +2,17 @@
 
 public class API_PATH {
     public static final String API_PATH_EMPLOYEES = "api/employees";
-    public static final String API_PATH_REQUESTS = "api/requests";
     public static final String API_PATH_LOGIN = "api/login";
     public static final String API_PATH_GET_SUPERIORS = "api/superiors";
     public static final String API_PATH_GET_EMPLOYEE = "api/employees/{id}";
+
+    public static final String API_PATH_REQUESTS = "api/requests";
+    public static final String API_PATH_GET_REQUEST = "api/requests/{id}";
+    public static final String API_PATH_CHANGE_STATUS_REQUEST = "api/requests/change";
```

Comment:
better to make it clear, change "api/requests/change" to "api/requests/changeStatus"